src/Components/NavBar/NavBar.tsx

```diff
@@ -109,8 +97,8 @@ export const NavBar: React.FC<NavBarProps> = track<NavBarProps>({
                   Menu={NotificationsMenu}
                   Overlay={NotificationsBadge}
                   onClick={() => {
-                    tracking.trackEvent({
-                      subject: Schema.Subject.NotificationBell,
+                    tracking.trackEvent<{ new_notification_count: string }>({
```

Comment:
For one-off tracking props that aren't a part of the AnalyticsSchema we can pass a generic